testutil/network/types.go

```diff
@@ -0,0 +1,56 @@
+package network
+
+import (
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	"github.com/cosmos/cosmos-sdk/testutil/network"
+)
+
+// TestMerkleProofPath is intended to be used as a "path" in a merkle tree for
+// which to generate a proof.
+var TestMerkleProofPath = []byte("test_proof_merkle_path")
+
+// InMemoryNetworkConfig is a **SHARED** config struct for use by InMemoryCosmosNetwork
+// implementations to configure themselves, provide the necessary parameters to set-up
+// code, and initialize the underlying cosmos-sdk testutil network.
+//
+// Examples of set-up operations include but are not limited to:
+// - Creating accounts in the local keyring.
+// - Creating genesis state for (a) module(s).
+// - Executing on-chain transactions (i.e. on-chain, non-genesis state).
+type InMemoryNetworkConfig struct {
+	// NumSessions is the number of sessions (with sequential start heights) for
+	// which the network should generate claims and proofs.
+	NumSessions int
+
+	// NumRelaysPerSession is the number of nodes to be inserted into each claim's
+	// session tree prior to generating the corresponding proof.
+	NumRelaysPerSession int
```

Comment:
I think the answer to your question is that it would be network-specific. Usage of field is per-claim, so it depends on what the network implementation's setup does in terms of claim creation.